README.md

```diff
@@ -61,6 +63,35 @@ in the root of the `meteor` repository. The typical symptom of not
 updating submodules will be `Error: Depending on unknown package ...`
 when you run most Meteor commands.
 
+### Create testing app
+
+To create a local app to test with your Meteor checkout this is a good 
+structure:
+
+```
+./meteor
+./demo-app
+```
+
+Create the demo app by running:
+
+```bash
+./meteor/meteor create demo-app
+```
+
+To run the demo-app with the local copy of Meteor:
+
+```bash
+cd demo-app
+../meteor/meteor
+```
+
```

Comment:
I still propose getting rid of this `### Create testing app` section.  The steps above this point still aren't clear and it's not really typical for someone to create a testing app in this fashion when they're developing Meteor features.  They probably got here in the first place because they're working with an existing app and they ran into a bug or wanted a feature.

Let's consider just giving better instructions on how to use this with existing apps (as I posted [here](https://github.com/meteor/meteor/pull/7823#discussion_r80711575))